doc/modules/classes.rst

```diff
@@ -143,43 +143,58 @@ Classes
    covariance.oas
    covariance.graph_lasso
 
+.. _model_selection_ref:
```

Comment:
I'm +0.5 about keeping the "old and will be removed". It might hurt googlability so we need good links to the new non-deprecated files.